deploy/routeros.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
do not use bash

Reply:
changed to /usr/bin/env sh (same as in deploy/gitlab)